pyproject.toml

```diff
@@ -58,7 +61,6 @@ numpy = "^1.24.2"
 [tool.poetry.group.dev.dependencies]
 pytest = ">=7.1.0"
 flake8 = "4.0.1"
-pylint = "2.12.2"
```

Comment:
pylint was holding python 3.11 back in poetry: wrapt 1.13 deprecated and will fail with pylint 2.12